django_access_logger/tests/test.py

```diff
@@ -0,0 +1,53 @@
+from django.test import TestCase
+import logging
+# Let's try to do this: https://github.com/selwin/django-user_agents/blob/master/django_user_agents/tests/urls.py
```

Comment:
Do you mean to support Django versions prior to 1.8?

Reply:
No, I didn't mean to commit this part, it was just a working example of a middleware lib with this kind of testing